doc/agent.rst

```diff
@@ -61,6 +63,14 @@ Below is a sample heartbeat message from a linux agent stored in
 		"version": "201407310027+bcbdd94.prod"
 	}
 
+It's possible on an endpoint information in the environment will change. For
```

Comment:
It's possible that information on an endpoint will change depending on the environment ? (please rephrase)